server/server/server.go

```diff
@@ -116,7 +118,13 @@ func New(cfg *config.Configuration, singleUserToken string, logger *mlog.Logger)
 	}
 	metricsService := metrics.NewMetrics(instanceInfo)
 
-	appServices := app.AppServices{
+	// Init audit
```

Comment:
As of now, auditing is a mandatory thing and the server fails to launch without `audit.json`. Is this intentional?

Reply:
I don't think so. If the `cfg.AuditFile` and `cfg.AuditJSON` are both empty the configuration still succeeds and the audit calls don't do anything.

Reply:
I've removed the sample config to make this more clear.

Reply:
I also removed the sample logger config and added a default one.  Thanks for raising this - the default logger was needed or zero logging would happen by default.

Reply:
Awesome!